Dockerfile

```diff
@@ -38,22 +33,19 @@ RUN yum update -y && yum install -y deltarpm && \
                     java-11-openjdk \
                     java-11-openjdk-devel \
                     lbzip2 \
-                    lbzip2-utils \
                     libffi \
                     libffi-devel \
                     libX11-devel \
                     libxml2-devel \
                     libxslt-devel \
-                    lockfile-progs \
```

Comment:
I checked the docu on about the prerequisites here again:

https://confluence.alma.cl/display/ICTACS/ACS+Prerequisites

And could not find any `lockfile-progs` so I just removed it.